R/checks.R

```diff
@@ -383,7 +396,7 @@ multihost_checks <-
     names(outs) <- failed_check_list
     return(outs)
   }
-}
+  }
```

Comment:
this looks like it is misalligned shouldn't have the tab there.

Reply:
removed in latest commit